packages/manager/apps/cloud/package.json

```diff
@@ -26,6 +26,7 @@
     "@ovh-ux/manager-banner": "^1.1.3",
     "@ovh-ux/manager-config": "^0.4.0",
     "@ovh-ux/manager-core": "^8.0.0 || ^9.0.0",
+    "@ovh-ux/manager-dbaas-logs": "^0.0.0 || ^1.0.0",
```

Comment:
I don't see any `BREAKING CHANGE` commit which will allow to release a `1.0.0` for the module

Reply:
The version has been updated to `0.1.0`. Thanks